### ISSUE TITLE
When hiding boosts from a user there is no notification when they boost your posts

### ISSUE THREAD

**Issue by chriswmartin** [tootsuite/mastodon#6156]
While using the new boost hiding feature (#5762) to remove a user's boosts from my home timeline it still seems like expected behavior to me that I would receive a notification when they boost something I posted, this isn't currently the case.

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Gargron** [comment]
Afaik that feature/PR does not touch NotifyService??

**Comment by chriswmartin** [comment]
If I understand correctly I think it does here https://github.com/tootsuite/mastodon/blob/master/app/services/notify_service.rb#L31

**Comment by Gargron** [comment]
Oof, you're right. I missed that. I see no reason why that should be part of the feature... cc @aschmitz

**Comment by aschmitz** [comment]
Yeah, I don't see a problem making boosts of your posts show up as notifications, I was probably just too thorough in not telling users about boosts of boost-muted users' boosts. (Although *technically* it's working as described, it's probably appropriate to show the boost as long as the user boosting isn't muted entirely.)
